market-dapp/backend/migrations/2_deploy_ipfscontract.js

```diff
@@ -1,5 +1,5 @@
 var STMarketplace = artifacts.require("./STMarketplace.sol");
 
 module.exports = function(deployer) {
-  deployer.deploy(STMarketplace);
+  deployer.deploy(STMarketplace, '0x5660a2Ead35A975D9F7C96d0B5f5FA12A12710d8');
```

Comment:
Ok, later on we will change this `0x5660..` address for the real deployed Descartes address